cmd/root.go

```diff
@@ -73,16 +87,19 @@ var RootCmd = &cobra.Command{
 		}
 
 		ui := &UI{
-			Owner:      owner,
-			Repository: repository,
+			Owner:      repo.Owner,
+			Repository: repo.Name,
+			Host:       repo.Host,
 			StartDate:  startDate,
 			EndDate:    endDate,
 			Query:      query,
 			CSVFormat:  csvFormat,
 			Calendar:   calendar,
 		}
 
-		fmt.Println(ui.PrintMetrics())
+		cmd.Println(ui.PrintMetrics())
```

Comment:
I believe `fmt.Println` hides the output from the `root_test` tests, such that successful output was not visible to those tests, which assume output is sent to a custom `new(bytes.Buffer)`. By using `cmd.Println`, the output is now successfully sent to that `bytes.Buffer` where the tests can make assertions on its value.

Reply:
I wasn't previously aware of it either, honestly. I happen to discover it when attempting to validate my changes :)